device/android/.gitignore

```diff
@@ -0,0 +1,12 @@
+*.iml
```

Comment:
I suggest that we use `/android`, not `/device/android`.

What are you going to put inside `/device` in addition to `android`?

Reply:
I thought we might have IOS or other systems in the future.